File: ruby/lib/card.rb
class Card 
	attr_reader :color, :label, :move_count
	def initialize(color,label,count)
		@color =  color
		@label = label
		@move_count = count
	end

	def to_string
		if @move_count == 1 && @color != "Pink"
			return "Single #{@color}"
		elsif @move_count == 2
			return "Double #{@color}"
		else
			return "#{@label}"
		end
	end
end

File: python/card.py
class Card(object):
	def __init__(self, color,label,move_count):
		#super(Card, self).__init__()
		self.color = color
		self.label = label
		self.move_count = move_count

	def to_string(self):
		if self.move_count == 1 and self.color != "Pink":
			return "Single %s" % self.color
		elif self.move_count == 2:
			return "Double %s" % self.color
		else:
			return "%s" % self.label


		